conf/variant/intel/bblayers.conf.sample

```diff
@@ -0,0 +1,12 @@
+# These are needed in order to make bitbake pick up the includes
+BSPDIR := "${TOPDIR}/../"
+BBPATH .= ":${BSPDIR}/sources/meta-pelux"
```

Comment:
I see these identical definitions in other places; maybe they could also be moved to the common conf?

Reply:
Unfortunately not, since BBPATH is the variable used for searching for the common conf. Hence it is needed in the top of any file that is a top-level bblayers.conf (which then can pull in common stuff)